mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/authentication.feature

```diff
@@ -1,7 +1,38 @@
+@ignore
 @parallel=false
 Feature: Authentication
 
+  Background:
+    * url baseUrl
 
-  @Undefined
-  Scenario: Authentication
-    * print 'Authentication'
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json'  }
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * print 'Create central servers'
+    * callonce read(featuresPath + 'central-server.feature@create')
+
+  Scenario: Successful authentication
+
+    * print 'Successful authentication'
+    Given path '/inn-reach/authentication'
+    * def authReq = read(samplesPath + "/authentication/authentication-request.json")
+    And request authReq
```

Comment:
`And request read(samplesPath + "/authentication/authentication-request.json")`

Reply:
refactored